src/Analysis/Engine/Impl/ModuleAnalysis.cs

```diff
@@ -114,7 +114,9 @@ public sealed class ModuleAnalysis : IModuleAnalysis {
         }
 
         internal IEnumerable<AnalysisVariable> ToVariables(IReferenceable referenceable) {
-            if (referenceable is VariableDef def) {
+            var def = referenceable as VariableDef;
```

Comment:
Why pattern matching didn't work here?

Reply:
Below code didnt use def before and now it does

Reply:
Ok